src/components/ui/Input.tsx

```diff
@@ -71,26 +67,43 @@ const inputContainer = (
       'focus:text-formal-accent'
     )
   )
+const iconStyles = classnames(height('h-full'), margin('mr-2'))
 
 export default function ({
   value,
+  valueList,
+  removeValueFromList,
   leftIcon,
   isError,
   disabled,
   ...rest
 }: {
   leftIcon?: ComponentChildren
   value?: string
+  valueList?: EmailMapping
+  removeValueFromList?: (fileName: string, index: number) => void
   isError?: boolean
   disabled?: boolean
 } & HTMLAttributes<HTMLInputElement>) {
   return (
     <div className={groupContainer(isError, disabled)}>
-      {leftIcon && <div className={iconContainer}>{leftIcon}</div>}
+      {leftIcon && <div className={iconStyles}>{leftIcon}</div>}
+      {removeValueFromList &&
+        valueList &&
+        Object.keys(valueList).map((fileName) =>
```

Comment:
We need the filename here, but `Object.entries` will look too complicated, so I separated it into two maps